arangod/Aql/SortNode.cpp

```diff
@@ -21,22 +21,38 @@
 /// @author Max Neunhoeffer
 ////////////////////////////////////////////////////////////////////////////////
 
-#include "SortNode.h"
 #include "Aql/Ast.h"
 #include "Aql/ExecutionPlan.h"
 #include "Aql/SortBlock.h"
 #include "Aql/WalkerWorker.h"
 #include "Basics/StringBuffer.h"
+#include "Basics/VelocyPackHelper.h"
+#include "SortNode.h"
+
+namespace {
+std::string const ConstrainedHeap = "ConstrainedHeap";
```

Comment:
can we use lower-case names for the strategies? then it would be in line with all the other things we report back for the plan in VelocyPack (e.g. for the GatherNode, which also uses a strategy, etc.)

Reply:
So do you think we should use kebab-case, e.g. "constrained-heap"?